e2e/cypress/integration/system_console/sidebar_link_navigation_spec.js

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('System Console', () => {
+    before(() => {
+        const newSettings = {
+            TeamSettings: {SiteName: 'Mattermost'},
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        // # Login as System Admin
+        cy.apiLogin('sysadmin');
+
+        cy.visit('/');
+    });
+
+    it('can go to admin console by clicking System Console', () => {
+        cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+
+        cy.get('.Menu__content').should('be.visible').within((container) => {
+            cy.getByText(/System Console/, {container}).should('be.visible').click();
+        });
+
+        cy.url().should('include', '/admin_console/about/license');
```

Comment:
@prapti I couldn't reproduced the one you're seeing where above test failed.  "System Console" has href of `/admin_console` which redirects to `/admin_console/about/license` on click.  How about let's keep the above check as is and see if the same will fail on daily test?

Reply:
Sounds good @saturninoabril. Thank you for following up on the error!